OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:01eeb933-3210-4eee-975c-103720fd86fd",
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "01eeb933-3210-4eee-975c-103720fd86fd",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.125+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Procedure"
+          ]
+        },
+        "status": "preparation",
+        "code": {
+          "coding": [
+            {
+              "code": "64444005",
+              "system": "http://snomed.info/sct",
+              "display": "Flow cytometry"
+            }
+          ],
+          "text": "Bone marrow or peripheral blood flow cytometry"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "report": [
+          {
+            "reference": "urn:uuid:67316637-1ed2-4258-abfa-9ddcba8522c1",
+            "type": "DocumentReference"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:67316637-1ed2-4258-abfa-9ddcba8522c1",
+      "resource": {
+        "resourceType": "DocumentReference",
+        "id": "67316637-1ed2-4258-abfa-9ddcba8522c1",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.136+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/DocumentReference"
+          ]
+        },
+        "status": "current",
+        "type": {
+          "coding": [
+            {
+              "code": "64444005",
+              "system": "http://snomed.info/sct",
+              "display": "Flow cytometry"
+            }
+          ],
+          "text": "Bone marrow or peripheral blood flow cytometry report"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "content": [
+          {
+            "attachment": {
+              "contentType": "application/pdf",
+              "data": "Qm9uZSBtYXJyb3c=",
+              "title": "Bone marrow or peripheral blood flow cytometry"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:571dc542-1afc-4981-94f3-5a54f198ab00",
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "571dc542-1afc-4981-94f3-5a54f198ab00",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.136+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Procedure"
+          ]
+        },
+        "status": "preparation",
```

Comment:
if the procedure has a report, status can not be preparation. Mark status as completed. Also, give representative examples - cytogenetics tests are not going to be available as part of OP Consultation. Such reports will come only as Diagnostic Results. Only minor procedures or those which can be done in consultation settings will come as part of the procedure section. Medication History section can have past procedure results